src/cf.data.pre

```diff
@@ -2570,7 +2570,16 @@ DOC_START
 	Allows you to apply a Netfilter mark value to packets being transmitted
 	on the client-side, based on an ACL.
 
-	clientside_mark mark-value [!]aclname ...
+	clientside_mark mark[/mask] [!]aclname ...
+
+	Mark and mask are unsigned integers (hex, octal, or decimal).
+	The mask may be used to preserve the mark previously set by other agents
```

Comment:
s/the mark/marking/
or
s/the mark/portions of the mark/
or
s/the mark/mark bits/